regulations/templates/regulations/preamble-header.html

```diff
@@ -1,9 +1,27 @@
 <div class="preamble-header">
-  <span>Actions:</span>
+  <span class="tab-descriptions">Actions:</span>
   <div class="read-proposal active-mode">
-    Read the proposal
+    <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+    <svg width="12px" height="11px" viewBox="0 0 12 11" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+        <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+            <g class="read-icon">
+                <g>
+                    <path d="M11,2.9 C11.2,3.2 11.2,3.4 11.1,3.8 L9.3,9.9 C9.2,10.2 9,10.4 8.8,10.6 C8.5,10.8 8.3,10.9 8,10.9 L1.8,10.9 C1.5,10.9 1.1,10.8 0.8,10.5 C0.5,10.3 0.3,10 0.1,9.6 C-2.84217094e-14,9.3 -2.84217094e-14,9 0.1,8.7 L0.1,8.5 L0.1,8.3 L0.1,8.2 L0.1,8.1 C0.1,8.1 0.1,8 0.2,8 C0.2,8 0.3,7.9 0.3,7.8 C0.3,7.7 0.4,7.7 0.4,7.6 C0.5,7.4 0.6,7.2 0.7,7 C0.8,6.8 0.9,6.6 0.9,6.4 L0.9,6.2 L0.9,6 C0.9,6 1,5.9 1,5.8 C1,5.7 1.1,5.7 1.1,5.6 C1.2,5.4 1.3,5.2 1.4,5 C1.5,4.8 1.5,4.5 1.6,4.4 L1.6,4.2 L1.6,4 C1.6,3.9 1.7,3.9 1.7,3.8 C1.8,3.7 1.8,3.7 1.8,3.6 C1.9,3.5 2,3.3 2.1,3 C2.2,2.7 2.3,2.5 2.3,2.4 L2.3,2.2 L2.3,2 C2.3,2 2.3,1.9 2.4,1.9 C2.4,1.9 2.5,1.8 2.5,1.7 C2.5,1.6 2.6,1.6 2.6,1.6 C2.6,1.5 2.7,1.5 2.7,1.4 C2.7,1.3 2.8,1.2 2.8,1.2 L2.9,1 C2.9,0.9 3,0.8 3,0.8 C3,0.7 3.1,0.7 3.2,0.6 C3.3,0.5 3.4,0.5 3.4,0.5 L3.7,0.5 L3.7,0.5 C3.9,0.5 4,0.4 4,0.4 L9.1,0.4 C9.4,0.4 9.7,0.5 9.9,0.8 C10.1,1 10.1,1.3 10,1.7 L8.2,7.8 C8,8.3 7.9,8.7 7.7,8.8 C7.5,9 7.3,9 6.8,9 L1,9 C0.9,9 0.8,9 0.7,9.1 C0.7,9.2 0.6,9.3 0.7,9.4 C0.8,9.7 1.1,9.9 1.7,9.9 L7.9,9.9 C8,9.9 8.2,9.9 8.3,9.8 C8.4,9.7 8.5,9.6 8.5,9.5 L10.5,2.9 L10.5,2.5 C10.8,2.7 10.9,2.8 11,2.9 L11,2.9 Z M3.3,4.6 L3.3,4.8 L3.4,4.9 L7.5,4.9 C7.6,4.9 7.6,4.9 7.7,4.8 C7.8,4.8 7.8,4.7 7.8,4.6 L7.9,4.2 L7.9,4 L7.8,3.9 L3.7,3.9 C3.6,3.9 3.6,3.9 3.5,4 C3.4,4 3.4,4.1 3.4,4.2 L3.3,4.6 L3.3,4.6 Z M3.9,3.1 L4,3.2 L8.1,3.2 C8.2,3.2 8.2,3.2 8.3,3.1 C8.4,3.1 8.4,3 8.4,2.9 L8.5,2.5 L8.5,2.3 L8.4,2.2 L4.3,2.2 C4.2,2.2 4.2,2.2 4.1,2.3 C4,2.3 4,2.4 4,2.5 L3.9,2.9 L3.9,3.1 Z" id="Shape"></path>
+                </g>
+            </g>
+        </g>
+    </svg>
```

Comment:
Would it make sense to use the icon font here, if we're already modifying this chunk?